### PR TITLE
Adds the Display Layer View State sample

### PR DESCRIPTION
**Pull request by philium**

### CODE REVIEW

File: arcgis-runtime-samples-macos/Maps/Display layer view state/DisplayLayerViewStateViewController.swift [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        map.operationalLayers.addObjects(from: [tiledLayer, imageLayer, featurelayer])
+        
+        return map
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        mapView.map = makeMap()
+        mapView.setViewpoint(AGSViewpoint(center: AGSPoint(x: -11e6, y: 45e5, spatialReference: .webMercator()), scale: 5e7))
```
Comment (nixta) [inline arcgis-runtime-samples-macos/Maps/Display layer view state/DisplayLayerViewStateViewController.swift]: Can I ask why you're using `-11e6` and `45e5` here? I think it's especially weird with the scale. I don't think anyone says "my map scale is 1:5e7".

Especially when above, in `makeMap()` you're using `40000000` and `2000000` for `minScale` and `maxScale`.

I would prefer the coordinates and scale be written out in full.

Comment (philium) [reply arcgis-runtime-samples-macos/Maps/Display layer view state/DisplayLayerViewStateViewController.swift]: > Can I ask why you're using -11e6 and 45e5 here?

I just copied what the iOS implementation had.

**Comment by philium** [comment]
@qsmmorgan I didn't add `LayerViewStatusPanel.swift` to the source file list because it doesn't contain any runtime-specific code. See the distance measurement analysis sample for a similar approach.

**Comment by qsmmorgan** [comment]
Up to you, but I'd have included it.

**Pull request merged**